finat/sympy2gem.py

```diff
@@ -65,3 +65,48 @@ def sympy2gem_symbol(node, self):
 @sympy2gem.register(symengine.Rational)
 def sympy2gem_rational(node, self):
     return gem.Division(*(map(self, node.as_numer_denom())))
+
+
+@sympy2gem.register(sympy.Abs)
+@sympy2gem.register(symengine.Abs)
+def sympy2gem_abs(node, self):
+    return gem.MaxValue(*map(self, node.args), gem.Literal(0))
```

Comment:
Surely not?!

Reply:
There's no `gem.Abs`, or I am wrong?

Reply:
Use `gem.MathFunction("abs", *map(self, node.args))`, but more concretely `max(-1, 0) != abs(-1)` !